s-mode-utils/src/sbi_console.rs

```diff
@@ -22,13 +21,7 @@ impl SbiConsole {
 impl ConsoleWriter for SbiConsole {
     /// Write an entire byte sequence to the SBI console.
     fn write_bytes(&self, bytes: &[u8]) {
-        for &b in bytes {
-            let message = SbiMessage::PutChar(b as u64);
-            // Safety: message doesn't contain pointers and the ecall doesn't touch memory so this
-            // is trivially safe.
-            unsafe {
-                ecall_send(&message).unwrap();
-            }
-        }
```

Comment:
Note that this will break early boot prints from Salus unless OpenSBI supports the DBCN. (There's a branch for that here: https://github.com/avpatel/opensbi/tree/riscv_sbi_dbcn_v1)